### DISCUSSION TITLE
call_fp

### DISCUSSION THREAD

**Discussion by pierrec** [maximecb/uvm#39] [Q&A]
Hello @maximecb,

I see that there is a commented entry for `call_fp`. Is there a plan to implement it? I would make use of it to implement interfaces!

**Comment by maximecb** [comment]
Yes this is certainly possible. I'll try to find time to do it tonight.

**Reply by pierrec** [reply]
thank you!

**Comment by maximecb** [comment]
`call_fp` is now implemented: https://github.com/maximecb/uvm/commit/aef97d440c4db5f87d0b7b8e043c6f218e0942a6

:)

**Reply by pierrec** [reply]
Eager to try it out!
